packages/mdc-drawer/package-lock.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
I think you previously removed this on the other branch? Not sure if that was with the intent of adding it here or removing it altogether.